src/cueTextTracks.js

```diff
@@ -7,14 +7,16 @@
 
 import videojs from 'video.js';
 
+const cueTextTracks = {};
```

Comment:
This file contains the only meaningful source change. The reason is that Rollup properly makes export bindings constant; so, when doing `import * as foo from '...';`, the properties of `foo` are read-only. This is how ES6 modules are supposed to work.

But contrib-ads exposes `cueTextTracks` and overrides properties of it in tests, which would throw errors with the old code because they became read-only properties.